api/db-cdr/generate-cdr/validate-prerequisites-exist.sh

```diff
@@ -95,6 +108,15 @@ if gsutil -m cp gs://$BUCKET/$BQ_DATASET/$CSV_HOME_DIR/*.csv $TEMP_FILE_DIR
       bq load --project_id=$BQ_PROJECT --source_format=CSV $BQ_DATASET.$tableName gs://$BUCKET/$BQ_DATASET/$CSV_HOME_DIR/$file $schema_path/$tableName.json
       echo "Finished loading $file"
     ;;
+    $DATA_DICTIONARY_FILE)
+      tableName=${file%.*}
+      #zip file
+      gzip $TEMP_FILE_DIR/$file
+
+      echo "Copy file to the version folder"
+      gsutil mv $TEMP_FILE_DIR/ds_data_dictionary.csv.gz gs://$BUCKET/$BQ_DATASET/$CDR_VERSION/ds_data_dictionary.csv.gz
+      echo "Finished loading $file"
+    ;;
```

Comment:
I kept it as a separate Case since the ds_data_dictionary does not have any header. I'll update this and change the current csv files to include header in the bucket

Reply:
Doesn't matter if it has a header or not.. it will work for either.

Reply:
What i have seen in testing, if it doesnt have a header it deletes the first row

Reply:
Done